### PR TITLE
High level architecural drawing + navigation

### PR DESCRIPTION
**Pull request by pmatulis**
Architectural overview is not finished. This is just a piece.

### CODE REVIEW

File: src/en/intro-architecture.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Title: MAAS | Architectural Overview
+
+
+# Architectural overview
+
+The high level architecture of MAAS is presented below.
+
+![intro-arch-overview-1](../media/intro-arch-overview-1.png)
```
Comment (degville) [inline src/en/intro-architecture.md]: Looks good - tiny thing, I've been using './media' in my docs.

**Comment by degville** [comment]
Looks good - just added a note to say I've been using ./media to reference images in my docs.

**Comment by pmatulis** [comment]
The . works for you because in the htmldocs directory someone put a symbolic link to the actual 'media' directory one level up:

$ ls
lrwxrwxrwx 1 pmatulis pmatulis   12 2016-06-08 17:22 resources -> ../resources/
lrwxrwxrwx 1 pmatulis pmatulis    8 2016-06-08 17:22 media -> ../media/
drwxrwxr-x 2 pmatulis pmatulis 4.0K 2016-06-08 17:22 js/
drwxrwxr-x 2 pmatulis pmatulis 4.0K 2016-06-08 17:22 css/
drwxrwxr-x 2 pmatulis pmatulis 4.0K 2016-06-14 13:04 en/

So why is the media directory not parallel to the htmldocs directory (like it is with the Juju docs)? Ditto for the 'resources' directory.

We should definitely use a standard and it makes too much sense to use the structure we presently have with Juju.

@evilnick

**Comment by degville** [comment]
I put the link there to keep paths consistent with Juju docs, deferring a decision on media and resources until things were working.

**Comment by pmatulis** [comment]
Ok, I'll use a single dot going forward.

**Pull request merged**